app/app.go

```diff
@@ -258,6 +254,8 @@ func NewApplication(
 		}
 	}
 
+	// setup postHandler in this method
+	// app.setupPostHandler()
```

Comment:
why it's commented?

Reply:
There is currently no logic in the post handler.
Just defined the method in case we use in future.